packages/server/graphql/public/typeDefs/Organization.graphql

```diff
@@ -156,6 +156,11 @@ type Organization {
   """
   featureFlags: OrganizationFeatureFlags!
 
+  """
+  Has access to AI features
+  """
+  hasAI: Boolean!
```

Comment:
curious, why put this on the organization when it is defined at the environment level?
could we instead send this down as property on `window.__ACTION__` like we do with other environment vars?

Reply:
Ahh yes, of course! I like that approach. A lot less code. Updated!